lib/suspenders/generators/analytics_generator.rb

```diff
@@ -0,0 +1,29 @@
+require "rails/generators"
+
+module Suspenders
+  class AnalyticsGenerator < Rails::Generators::Base
+    source_root File.expand_path(
+      File.join("..", "..", "..", "templates"),
+      File.dirname(__FILE__),
+    )
+
+    def install_partial
+      copy_file "_analytics.html.erb",
+        "app/views/application/_analytics.html.erb"
+    end
+
+    def render_partial
+      if File.exist?(js_partial)
```

Comment:
Curious when this view would not exist. In the prior PR, it was always copied.

```
def create_shared_javascripts
  copy_file "_javascript.html.erb",
    "app/views/application/_javascript.html.erb"
end
```

Is there some conditional I'm missing?

Reply:
Now that this is an independent task I tried to consider what it'd be like if it ran in isolation from the other tasks. If the `suspenders:views` generator isn't run, the `_analytics.html.erb` template won't exist and this will error.

Reply:
Makes sense!